src/main.ts

```diff
@@ -1267,8 +1267,8 @@ export class FileBrowserFeature extends SqlOpsFeature<undefined> {
 	protected registerProvider(options: undefined): Disposable {
 		const client = this._client;
 
-		let openFileBrowser = (ownerUri: string, expandPath: string, fileFilters: string[], changeFilter: boolean): Thenable<boolean> => {
-			let params: types.FileBrowserOpenParams = { ownerUri, expandPath, fileFilters, changeFilter };
+		let openFileBrowser = (ownerUri: string, expandPath: string, fileFilters: string[], changeFilter: boolean, showFoldersOnly?: boolean): Thenable<boolean> => {
```

Comment:
Would suggest updating azdata.d.proposed.ts with this as well and then bringing the updated file into this PR

Reply:
How does that work if the original method definition is in the stable azdata.d.ts file? https://github.com/microsoft/azuredatastudio/blob/main/src/sql/azdata.d.ts#L2276

Reply:
You create another interface and define the new function definition there. Typescript will combine them into one for purposes of typing though (similar to an overload, but there's only one actual definition, this is allowed as long as you only add new properties)